doc-content/enterprise-only/employee-rostering/er-contracts-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='er-contracts-proc']
+= Entering the list of contracts
+
+You must enter the list of all types of contracts that the business uses for employees.
+
+A contract determines the maximum time that the employee can work in a day, calendar week, calendar month, or calendar year.
+
+You can set any of the limitations or none at all is a contract. For example, a part-time employee might not be allowed to work more than 20 hours per week, while a full-time employee might be limited to 10 hours in a day and 1800 hours during a year. Another contract might include no limitations on worked hours.
+
+You must enter all work time limits for contracts in minutes.
+
```

Comment:
Any prereqs? Do I do this on paper :-) Something like:
Prerequisites
You have an employee rostering  application that you want to update.

Reply:
In a user guide to an application, do we really give a prereq that the application must be started? Would you have any examples in existing RH docs? For example, in our Business Central docs, I see no prereqs in a procedure that you start in the main menu of BC, such as https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.3/html/designing_a_decision_service_using_guided_rule_templates/guided-rule-templates-create-proc

So I really don't see why I should add "prerequisites - you have an application..." to all my  procedures in Employee Rostering.

The restructure of the OpenShift docs was different. There, one needed to be in a very particular spot in the UI. Here, one just needs to have the app running - you can get into any tab from any place. And judging by our existing examples we don't make "app is started" an explicit prereq. I suggest not doing this here either, if only for the same of consistency.

Reply:
OK, but you at least need to say where to start from, as in the example you provide above.
To enter or change the list of contracts, click the Contracts tab. Where do I do this?

Reply:
That's fine now.